### PR TITLE
Configure CI to run conformance tests

### PR DESCRIPTION
**Pull request by davidor**
This PR adds the scripts needed to run the conformance tests. I basically adapted the ones in net-istio and net-contour to our use case.

### CODE REVIEW

File: test/e2e-common.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  # Setting up test resources.
+  echo ">> Publishing test images"
+  $(dirname $0)/upload-test-images.sh || fail_test "Error uploading test images"
+  echo ">> Creating test resources (test/config/)"
+  ko apply ${KO_FLAGS} -f test/config/ || return 1
+
+  # Bringing up controllers.
+  echo ">> Bringing up Kourier"
+  ko apply -f deploy/ || return 1
+  wait_until_batch_job_complete kourier-system || return 1
```
Comment (markusthoemmes) [inline test/e2e-common.sh:42]: Is there even a job running here? I don't think this is necessary.

Comment (davidor) [reply test/e2e-common.sh:42]: I don't think it's needed :+1:

**Pull request merged**